website/docs/cloud-docs/api-docs/stacks.mdx

```diff
@@ -0,0 +1,2724 @@
+| `GET` | `/stack-events/:stack_event_id` | Call this endpoint to [show stack event](#show-stack-event).|
+| `GET` | `/stack-plans/:stack_plan_id/plan-description` | Call this endpoint to [show plan description](#show-plan-description).|
+| `GET` | `/stack-plans/:stack_plan_id/apply-description` | Call this endpoint to [show apply description](#show-apply-description).|
+| `GET` | `/stack-configurations/:stack_configuration_id/stack-plans` | Call this endpoint to [list stack configuration plans](#list-stack-configuration-plans).|
+| `GET` | `/stacks/:stack_id/stack-states` | Call this endpoint to [list stack states](#list-stack-states).|
+| `GET` | `/stack-states/:stack_state_id` | Call this endpoint to [show stack states](#show-stack-state).|
+| `GET` | `/stack-states/:stack_state_id/description` | Call this endpoint to [show stack state description](#show-stack-state-description).|
+| `POST` | `/stacks` | Call this endpoint to [create a stack](#create-a-stack).|
+| `POST` | `/stack-diagnostics/:stack_diagnostic_id/acknowledge` | Call this endpoint to [acknowledge stack diagnostic](#acknowledge-stack-diagnostic).|
+| `POST` | `/stacks/:stack_id/stack-deployments/:deployment_name/create-destroy-plan` | Call this endpoint to [create destroy plan](#create-destroy-plan).|
+| `POST` | `/stacks/:stack_id/stack-deployments/:deployment_name/delete-deployment` | Call this endpoint to [delete deployment](#delete-deployment).|
+| `POST` | `/stacks/:stack_id/actions/update-configuration` | Call this endpoint to [fetch a stack configuration](#fetch-the-latest-stack-configuration).|
+| `POST` | `/stacks/:stack_id/delete` | Call this endpoint to [delete a stack](#safe-delete-stack).|
+| `POST` | `/stacks/:stack_id/force-delete` | Call this endpoint to [force delete a stack](#force-delete-stack).|
+| `POST` | `/stack-plans/:stack_plan_id/approve` | Call this endpoint to [approve a stack plan](#approve-stack-plan).|
+| `POST` | `/stack-plans/:stack_plan_id/discard` | Call this endpoint to [discard a stack plan](#discard-stack-plan).|
+| `POST` | `/stack-plans/:stack_plan_id/cancel` | Call this endpoint to [cancel a stack plan](#cancel-stack-plan).|
+| `POST` | `/stack-plans/:stack_plan_id/replan` | Call this endpoint to [replan stack plan](#replan-stack-plan).|
+| `PATCH` | `/stacks/:stack_id` | Call this endpoint to [update a stack](#update-a-stack).|
+
+## List stacks
+
+This endpoint lists stacks in the organization.
+
+`GET /organizations/:organization_name/stacks`
+
+-> **Note:** Stack read access is restricted to users or teams with read project access and the [organization API token](/terraform/cloud-docs/users-teams-organizations/api-tokens#organization-api-tokens).
+
+| Parameter            | Description                                             |
+| -------------------- | ------------------------------------------------------- |
+| `:organization_name` | The name of the organization to list the stacks of. |
```

Comment:
Something I've done before is list the query parameters section. We could list the valid `include` data and any search/filter/pagination. An example of the policy sets query parameter section: https://developer.hashicorp.com/terraform/cloud-docs/api-docs/policy-sets#query-parameters